docs/installation.rst

```diff
@@ -36,3 +32,25 @@ first using `venv <https://docs.python.org/3/library/venv.html>`_::
     $ source tsinfer-venv/bin/activate
     (tsinfer-venv) $ pip install tsinfer
     (tsinfer-venv) $ tsinfer --help
+
+.. _sec_installation_installation_problems:
+
++++++++++++++++++++++
+Installation problems
++++++++++++++++++++++
+
+
+There are situations (usually where the GSL libraries are not in the
```

Comment:
We shouldn't be installing msprime at all now though, right? So, GSL shouldn't come into it at all.